packages/office-addin-dev-settings/src/commands.ts

```diff
@@ -179,6 +179,23 @@ export async function getSourceBundleUrl(manifestPath: string) {
   }
 }
 
+async function getWebViewType(manifestPath: string) {
+  try {
+    const manifest = await readManifestFile(manifestPath);
+
+    validateManifestId(manifest);
+
+    const webViewString = await devSettings.getWebView(manifest.id!);
```

Comment:
```
const webViewType: WebViewType = await devSettings.getWebView(manifest.id!);
const webViewTypeString = toWebViewTypeString(webViewType);

console.log(webViewTypeString 
    ? `The web view type is set to ${webViewTypeString}.` 
     : `The web view type has not been set.`);
```

Reply:
Yes, this means that the commands converts the string to an enum and back to the string, but that's OK.